src/components/App.js

```diff
@@ -1,21 +1,32 @@
-import React, { PureComponent } from 'react';
+import React, { Component } from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
+
 import calculate from '../logic/calculate';
 
-class App extends PureComponent {
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: '0',
+      next: '0',
+      operation: '',
+    };
```

Comment:
By default, `total`, `next`, and `operation` should be set to null. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/app/stateful.png))